oggsound/src/OgreOggSoundManager.cpp

```diff
@@ -288,18 +288,22 @@ namespace OgreOggSound
 		bool deviceInList = false;
 		if(majorVersion >= 1 && minorVersion >= 1)
 		{
-			Ogre::LogManager::getSingleton().logMessage("*** --- AVAILABLE DEVICES --- ***");
+			Ogre::LogManager::getSingleton().logMessage("*** --- AVAILABLE DEVICES:");
 
 			// List devices in log and see if the sugested device is in the list
-			Ogre::StringVector deviceList = getDeviceList();
```

Comment:
I think you should keep the old version here. With your changes to getDeviceList, it should work as intended